thirdeye/thirdeye-frontend/app/helpers/manage-alert-utils.js

```diff
@@ -4,18 +4,17 @@ import moment from 'moment';
 /**
  * Handles types and defaults returned from eval/projected endpoints
  * @param {Number|String} metric - number or string like 'NaN', 'Infinity'
- * @param {Boolean} allowDecimal - should this metric be shown as whole number?
+ * @param {Boolean} isPercentage - shall we treat this as a % or a whole number?
  * @returns {Object}
  */
-export function formatEvalMetric(metric, allowDecimal = false) {
-  let shown = 'N/A';
-  if (!isNaN(metric)) {
-    if (allowDecimal) {
-      shown = (Number(metric) === 1 || Number(metric) === 0)
-        ? metric * 100
-        : (metric * 100).toFixed(1);
+export function formatEvalMetric(metric, isPercentage = false) {
+  const isWhole = Number(metric) % 1 === 0;
+  let shown = (metric === 'Infinity') ? metric : 'N/A';
+  if (isFinite(metric)) {
+    if (isPercentage) {
```

Comment:
nit: there are opportunities for DRY-er code here, for instance:  
```
const multiplier = isPercentage ? 100 : 1; 
const formattedNum = metric * multiplier;
return isWhole ? formattedNum : formattedNum.toFixed(1);
```

Reply:
improved in https://github.com/linkedin/pinot/pull/2351